base/set_metrics_dir

```diff
@@ -0,0 +1 @@
+export METRICS_DIR="$METRICS_ROOT_DIR/$HOSTNAME"
```

Comment:
We will have to think about a layer of control or permissions in the future.  How do we control not letting metrics being written when a volume is not mounted?